hibike/devices/YogiBear/ui.cpp

```diff
@@ -0,0 +1,173 @@
+#include "Arduino.h" //to get digital write, etc.
+
+#include "ui.h"
+#include "motor.h"
+#include "pid.h"
+#include "encoder.h"
+#include "current_limit.h"
+//Code here that helps deal with the human-readable UI.
```

Comment:
So this is code that's used when hibike is disabled? Is this something that's set at compile time?